imports/plugins/included/taxes-taxcloud/server/hooks/hooks.js

```diff
@@ -42,84 +37,85 @@ MethodHooks.after("taxes/calculate", (options) => {
         };
       }
 
-      // check if plugin is enabled and this calculation method is enabled
-      if (pkg && pkg.enabled === true && pkg.settings.taxcloud.enabled === true) {
-        if (!apiKey || !apiLoginId) {
-          Logger.warn("TaxCloud API Key is required.");
-        }
-        if (Array.isArray(cartToCalc.shipping) && cartToCalc.shipping.length > 0 && cartToCalc.items) {
-          const shippingAddress = cartToCalc.shipping[0].address;
+      if (!apiKey || !apiLoginId) {
+        Logger.warn("TaxCloud API Key is required.");
+      }
+      if (Array.isArray(cartToCalc.shipping) && cartToCalc.shipping.length > 0 && cartToCalc.items) {
+        const shippingAddress = cartToCalc.shipping[0].address;
 
-          if (shippingAddress) {
-            Logger.debug("TaxCloud triggered on taxes/calculate for cartId:", cartId);
-            const url = "https://api.taxcloud.net/1.0/TaxCloud/Lookup";
-            const cartItems = [];
-            const destination = {
-              Address1: shippingAddress.address1,
-              City: shippingAddress.city,
-              State: shippingAddress.region,
-              Zip5: shippingAddress.postal
-            };
+        if (shippingAddress) {
+          Logger.debug("TaxCloud triggered on taxes/calculate for cartId:", cartId);
+          const url = "https://api.taxcloud.net/1.0/TaxCloud/Lookup";
+          const cartItems = [];
+          const destination = {
+            Address1: shippingAddress.address1,
+            City: shippingAddress.city,
+            State: shippingAddress.region,
+            Zip5: shippingAddress.postal
+          };
 
-            // format cart items to TaxCloud structure
-            let index = 0;
-            for (const items of cartToCalc.items) {
-              // only processs taxable products
-              if (items.variants.taxable === true) {
-                const item = {
-                  Index: index,
-                  ItemID: items.variants._id,
-                  TIC: "00000",
-                  Price: items.variants.price,
-                  Qty: items.quantity
-                };
-                index += 1;
-                cartItems.push(item);
-              }
+          // format cart items to TaxCloud structure
+          let index = 0;
+          for (const items of cartToCalc.items) {
+            // only processs taxable products
+            if (items.variants.taxable === true) {
+              const item = {
+                Index: index,
+                ItemID: items.variants._id,
+                TIC: "00000",
```

Comment:
If this is the tax code, should we be pulling this from the product rather than hard-coding it here? I understand this is existing code but it seems like a simple fix.